src/clj_sqs_extended/aws/sqs.clj

```diff
@@ -129,10 +130,10 @@
      :or   {format :transit}}]
    (let [request (->> (serdes/serialize message format)
                       (SendMessageRequest. queue-url))]
-     ;; WATCHOUT: The group ID is mandatory when sending fifo messages.
-     (doto request (.setMessageGroupId group-id))
+     ;; group ID is mandatory when sending fifo messages
```

Comment:
@daisybytes the `WATCHOUT` seems redundant when it's on every comment. leaving a comment is implicitly a "watchout" already. let's reserve the markers for critical info.